src/generic/datavgen.cpp

```diff
@@ -3192,6 +3356,303 @@ int wxDataViewMainWindow::GetLineHeight( unsigned int row ) const
 }
 
 
+void RowRange::Add(const unsigned int idx)
+{
+    RangeEntity *rng = NULL;
+
+    size_t count = m_ranges.GetCount();
```

Comment:
This should be const, i.e.

    const size_t count = m_ranges.GetCount();

to avoid changing it accidentally later and to indicate that it won't be change to the reader.